### PR TITLE
strace: update page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by vitorhcl — APPROVED** [review]
LGTM, thanks!

**Comment by fazlearefin** [comment]
> In a next PR, or maybe in this one, we could add more mnemonics to the above examples, to keep the page consistent with this example

Done. Added mnemonics wherever possible.

**Review by gutjuri — APPROVED** [review]

File: pages/linux/strace.md [hunk 1]
```diff
@@ -3,19 +3,19 @@
...
 
-- Start tracing a specific process by its PID:
+- Start tracing a specific [p]rocess by its PID:
 
 `strace -p {{pid}}`
 
-- Trace a process and filter output by system call:
+- Trace a [p]rocess and filt[e]r output by system call:
 
 `strace -p {{pid}} -e {{system_call_name}}`
```
Comment (vitorhcl) [inline pages/linux/strace.md:12]:
```suggestion
`strace -p {{pid}} -e {{system_call1, system_call2,...}}`
```

Comment (fazlearefin) [reply pages/linux/strace.md:12]: There should be no space between `,` and `system_call2`

Comment (vitorhcl) [reply pages/linux/strace.md:12]: Oops, that space was accidental, fixed

**Review by kbdharun — APPROVED** [review]

**Pull request merged**